pages/blog/_slug.vue

```diff
@@ -150,16 +154,34 @@ export default {
   content: {
     liveEdit: false,
   },
-  async asyncData({ $content, params }) {
-    const article = await $content('blog', params.slug).fetch()
+  async asyncData({ params }) {
+    const response = await axios.get(
```

Comment:
I think there's a way in axios to pass params. you don't have to do ?something=dsf manually

Reply:
Okay, I will change this in all places..